WooCommerce/Classes/ViewRelated/Dashboard/Free Trial/FreeTrialSurvey/FreeTrialSurveyView.swift

```diff
@@ -0,0 +1,123 @@
+import SwiftUI
+
+final class FreeTrialSurveyHostingController: UIHostingController<FreeTrialSurveyView> {
+    init(viewModel: FreeTrialSurveyViewModel) {
+        super.init(rootView: FreeTrialSurveyView(viewModel: viewModel))
+        rootView.dismissAction = dismiss
+    }
+
+    @available(*, unavailable)
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureTransparentNavigationBar()
+    }
+
+    func dismiss() {
+        dismiss(animated: true)
+    }
+}
+
+struct FreeTrialSurveyView: View {
+    @ObservedObject private var viewModel: FreeTrialSurveyViewModel
+    var dismissAction: () -> Void = {}
+
+    init(viewModel: FreeTrialSurveyViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 40) {
+                Text(Localization.title)
+                    .fontWeight(.bold)
+                    .titleStyle()
+
+                VStack() {
+                    ForEach(viewModel.answers, id: \.self) { answer in
+                        if answer == .otherReasons {
+                            TextField(answer.text, text: $viewModel.otherReasonSpecified)
+                                .font(.body)
+                                .textFieldStyle(RoundedBorderTextFieldStyle(focused: false))
```

Comment:
💡  Should we support focus state for the field so that it can be highlighted when selected?